extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisDynamicOutputFormat.java

```diff
@@ -113,24 +133,26 @@ public DorisDynamicOutputFormat(DorisOptions option,
                     LINE_DELIMITER_DEFAULT);
         }
 
-        this.fieldNames = fieldNames;
-        this.jsonFormat = FORMAT_JSON_VALUE.equals(executionOptions.getStreamLoadProp().getProperty(FORMAT_KEY));
-        this.fieldGetters = new RowData.FieldGetter[logicalTypes.length];
-        for (int i = 0; i < logicalTypes.length; i++) {
-            fieldGetters[i] = createFieldGetter(logicalTypes[i], i);
+        //add column key
+        if (!streamLoadProp.containsKey(COLUMNS_KEY)) {
+            String columns = String.join(",", Arrays.stream(fieldNames).map(item -> String.format("`%s`", item.trim().replace("`", ""))).collect(Collectors.toList()));
```

Comment:
if using datastream api, fieldNames should be defined.

Reply:
Thank you for reminding me that I ignored the datastream submission method